Session 3/task3.js

```diff
@@ -0,0 +1,17 @@
+/*
+http://www.codewars.com/kata/closures-and-scopes/train/javascript
+*/
+function createFunctions(n) {
+  var callbacks = [];
+
+  for (var i=0; i<n; i++) {
+    
```

Comment:
use let and arrow function

Reply:
We did not cover ES6 topics when I completed this task. Will do it.